### PR TITLE
Fix fixture issue, reorganize test, small corrections

### PR DESCRIPTION
**Pull request by legan4ik**

### CODE REVIEW

**Comment by legan4ik** [comment]
https://github.com/msenin94/mk-post-deployment-checks/issues/64

File: mk_verificator/tests/test_migration.py [hunk 1]
```diff
@@ -44,3 +42,4 @@ def test_migration_for_all_nodes(nova_client, create_vm):
     for host in list_hosts:
         create_vm.live_migrate(host=host)
         checking_vm_host(nova_client, create_vm.id, host)
+    create_vm.delete()
```
Comment (msenin94) [inline mk_verificator/tests/test_migration.py]: why did you delete yield??? that's more useful than keep object deletion in "unsafe" code block
